### PR TITLE
DIV-5772 - added div dgs, fps and emca prod configs

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

Subscription, client ID and object ID: https://hmcts-reform.slack.com/archives/CN0CSTTSN/p1568621528009500

DIV-5772

### Change description ###

added div cms, cos, dgs, fps and emca prod configs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by hosainnet** [comment]
> Move DB details as well to dynamic ? like https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L6
> Also, if keyvaults are same in values.yaml, you can avoid overriding them

Yep, just pushed the postgres change. https://github.com/hmcts/div-case-orchestration-service/pull/484/commits/ccc293637f2bcd63865788d4b7528559a348d899

File: k8s/prod/common/divorce/div-cos.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    name: div-cos
+    version: 1.3.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
+      image: hmctspublic.azurecr.io/div/cos:prod-aed712be
+      environment:
+        IDAM_API_URL: "https://idam-api.prod.platform.hmcts.net"
```
Comment (timja) [inline k8s/prod/common/divorce/div-cos.yaml]:
```suggestion
        IDAM_API_URL: "https://idam-api.platform.hmcts.net"
```

File: k8s/prod/common/divorce/div-cms.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    version: 1.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
+      image: hmctspublic.azurecr.io/div/cms:prod-33c59844
+      environment:
+        IDAM_CASEWORKER_USERNAME: "DivCaseWorker@AAT.com"
+        IDAM_API_BASEURL: "https://idam-api.prod.platform.hmcts.net"
```
Comment (timja) [inline k8s/prod/common/divorce/div-cms.yaml]:
```suggestion
        IDAM_API_BASEURL: "https://idam-api.platform.hmcts.net"
```

File: k8s/prod/common/divorce/div-cos.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cos
+    version: 1.3.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
+      image: hmctspublic.azurecr.io/div/cos:prod-aed712be
+      environment:
```
Comment (hosainnet) [inline k8s/prod/common/divorce/div-cos.yaml]: Please compare the below with https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/variables.tf and https://github.com/hmcts/div-case-orchestration-service/blob/master/infrastructure/prod.tfvars

File: k8s/aat/common/divorce/div-cms.yaml [hunk 4]
```diff
@@ -14,35 +14,21 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: div-cms
-    version: 1.1.0
+    version: 1.2.0
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-cms.yaml]: The new version allows us to get rid of duplicates by replacing `-aat/-prod` with `{{ .Values.global.environment }}` and move the variables into the common `values.yaml` in each project

File: k8s/aat/common/divorce/div-cms.yaml [hunk 5]
```diff
@@ -14,35 +14,21 @@ spec:
...
           -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
       aadIdentityName: divorce
-      keyVaults:
-        "div":
-          resourceGroup: div
-          secrets:
-          - ccd-submission-s2s-auth-secret
-          - idam-secret
-          - draft-store-api-encryption-key
-          - idam-caseworker-password
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-cms.yaml]: This is now in `values.yaml` as well so it doesn't need to be here

File: k8s/prod/common/divorce/div-cms.yaml (hunk 2 above)
Comment (hosainnet) [reply k8s/prod/common/divorce/div-cms.yaml]: 👍

File: k8s/prod/common/divorce/div-cms.yaml [hunk 6]
```diff
@@ -23,7 +23,7 @@ spec:
       image: hmctspublic.azurecr.io/div/cms:prod-33c59844
       environment:
         IDAM_CASEWORKER_USERNAME: "DivCaseWorker@AAT.com"
```
Comment (timja) [inline k8s/prod/common/divorce/div-cms.yaml]: Is this right

File: k8s/perftest/common/divorce/div-cos.yaml [hunk 7]
```diff
@@ -14,30 +14,15 @@ spec:
...
+    version: 1.3.0
   values:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
       image: hmctspublic.azurecr.io/div/cos:prod-aed712be
-      ingressHost: div-cos-perftest.service.core-compute-perftest.internal
       environment:
         IDAM_API_URL: "https://idam-api.perftest.platform.hmcts.net"
```
Comment (timja) [inline k8s/perftest/common/divorce/div-cos.yaml]: is this still needed?

File: k8s/aat/common/divorce/div-dgs.yaml [hunk 8]
```diff
@@ -28,12 +28,7 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
       image: hmctspublic.azurecr.io/div/dgs:prod-d730ffbc
-      ingressHost: div-dgs-aat.service.core-compute-aat.internal
       environment:
-        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
-        EVIDENCE_MANAGEMENT_CLIENT_API_BASEURL: "http://div-emca-aat.service.core-compute-aat.internal"
-        IDAM_S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
-        PDF_SERVICE_BASEURL: "http://cmc-pdf-service-aat.service.core-compute-aat.internal"
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-dgs.yaml]: Those are now here https://github.com/hmcts/div-document-generator-client/blob/master/charts/div-dgs/values.yaml#L5

File: k8s/aat/common/divorce/div-dgs.yaml [hunk 9]
```diff
@@ -42,12 +37,6 @@ spec:
         JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
           -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
       aadIdentityName: divorce
-      keyVaults:
-        "div":
-          resourceGroup: div
-          secrets:
-          - div-doc-s2s-auth-secret
-          - docmosis-api-key
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-dgs.yaml]: https://github.com/hmcts/div-document-generator-client/blob/master/charts/div-dgs/values.yaml#L15

File: k8s/aat/common/divorce/div-emca.yaml [hunk 10]
```diff
@@ -28,22 +28,12 @@ spec:
...
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
       image: hmctspublic.azurecr.io/div/emca:prod-b547e922
-      ingressHost: div-emca-aat.service.core-compute-aat.internal
       environment:
         AUTH_PROVIDER_SERVICE_CLIENT_MICROSERVICE: divorce_document_generator
-        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
         IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
-        EVIDENCE_MANAGEMENT_HEALTH_URL: http://dm-store-aat.service.core-compute-aat.internal/health
-        EVIDENCE_MANAGEMENT_UPLOAD_FILE_URL: http://dm-store-aat.service.core-compute-aat.internal/documents
-        DOCUMENT_MANAGEMENT_STORE_URL: http://dm-store-aat.service.core-compute-aat.internal
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-emca.yaml]: https://github.com/hmcts/div-evidence-management-client-api/blob/master/charts/div-emca/values.yaml#L5

File: k8s/aat/common/divorce/div-emca.yaml [hunk 11]
```diff
@@ -28,22 +28,12 @@ spec:
...
         JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
           -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
       aadIdentityName: divorce
-      keyVaults:
-        "div":
-          resourceGroup: div
-          secrets:
-          - div-doc-s2s-auth-secret
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-emca.yaml]: https://github.com/hmcts/div-evidence-management-client-api/blob/master/charts/div-emca/values.yaml#L14

File: k8s/aat/common/divorce/div-fps.yaml [hunk 12]
```diff
@@ -28,9 +28,7 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
       image: hmctspublic.azurecr.io/div/fps:prod-5e4711bb
-      ingressHost: div-fps-aat.service.core-compute-aat.internal
       environment:
-        FEE_API_URL: "http://fees-register-api-aat.service.core-compute-aat.internal"
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-fps.yaml]: https://github.com/hmcts/div-fees-and-payments-service/blob/master/charts/div-fps/values.yaml#L5

**Pull request merged**